lib/internal/errors.js

```diff
@@ -499,6 +499,10 @@ E('ERR_CRYPTO_HASH_DIGEST_NO_UTF16', 'hash.digest() does not support UTF-16',
   Error);
 E('ERR_CRYPTO_HASH_FINALIZED', 'Digest already called', Error);
 E('ERR_CRYPTO_HASH_UPDATE_FAILED', 'Hash update failed', Error);
+E('ERR_CRYPTO_INCOMPATIBLE_KEY_FORMAT',
+  (format, reason = 'is incompatible with other options.') => {
+    return `The selected key encoding ${format} ${reason}.`;
+  }, Error);
```

Comment:
Seems like the errors always pass through both arguments. Therefore I recommend to switch this to:
```js
'The selected key encoding %s %s.'
```